webclient/src/Hacksite.vue

```diff
@@ -2,16 +2,19 @@
   <div id="app">
     <router-view></router-view>
     <InfoFooter />
+    <FormLoading />
```

Comment:
Is this referencing something that didn't get included?

Reply:
Ah yes I removed that piece, was part of early development for the spinner.